easybuild/tools/job/pbs_python.py

```diff
@@ -453,7 +455,7 @@ def info(self, types=None):
             return None
 
         # convert single type into list
-        if type(types) is str:
+        if isinstance(types, str):
```

Comment:
Better use the `is_string` from `easybuild.tools.py2vs3`?

We should probably deprecate `pbs_python`, Torque is pretty much dead anyway...

Reply:
Automatic change w/o behavior change. Can be done later